Sample/Direct/LocalNotification.Sample.iOS/AppDelegate.cs

```diff
@@ -31,5 +28,14 @@ public override bool FinishedLaunching(UIApplication app, NSDictionary options)
 
             return base.FinishedLaunching(app, options);
         }
+
+        public override void ReceivedLocalNotification(UIApplication application, UILocalNotification notification)
```

Comment:
Should this method be commented as it is only required by if iOS is version is less than 10 ?

Reply:
The method does not get called in ios 10+

Reply:
Thanks for updating the documentation.